spec/javascripts/components/intervention-spec.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-env jasmine, jquery */
+/* global GOVUK */
+
+describe('Intervention component', function () {
+  var FIXTURE
+
+  function loadInterventionComponent () {
+    var element = document.querySelector('[data-module="gem-intervention"]')
+    new GOVUK.Modules.Intervention(element).init()
+  }
+
+  beforeEach(function () {
+    spyOn(GOVUK.analytics, 'trackEvent')
+    spyOn(GOVUK.Modules, 'GemTrackClick').and.callFake(function () { this.start = function () {} })
+
+    FIXTURE = `
+     <section class="gem-c-intervention" role="region" data-module="intervention">
+        <h2 class="gem-c-intervention__title govuk-heading-s">
```

Comment:
For the purposes of this test all of the markup here doesn't seem necessary, as it doesn't do anything, am I right? Since all we're testing is the component loading and firing a tracking event, could most of the HTML here be removed?